api/src/types/mimetext.d.ts

```diff
@@ -0,0 +1,163 @@
+declare module 'mimetext' {
+    export interface MIMETextError extends Error {
```

Comment:
lgtm. could you add attribution in block comment from wherever you got it from, and add a note about why we've got this here as well please